### DISCUSSION TITLE
96. 카디널리티의 개념과 어떤 카디널리티일 때 인덱스를 사용하면 좋은지 말해주세요.

### DISCUSSION THREAD

**Discussion by jongmee** [jewelry-box/cs-interview#100] [General]
카디널리티의 개념과 어떤 카디널리티일 때 인덱스를 사용하면 좋은지 말해주세요.

**Comment by linirini** [comment]
카디널리티는 데이터베이스의 특정 열이나 속성이 가질 수 있는 고유한 값의 수를 의미한다.
고유한 정도가 높을수록 카디널리티가 높다고 말한다.
카디널리티가 높을수록 인덱스 사용이 효과적이다. 검색 시 인덱스를 통해 특정한 값을 빠르게 찾고, 집합을 좁힐 수 있다.
